packages/app-builder-lib/templates/nsis/include/allowOnlyOneInstallerInstance.nsh

```diff
@@ -30,22 +30,77 @@
 !macroend
 
 !macro CHECK_APP_RUNNING
-    !ifmacrodef customCheckAppRunning
-      !insertmacro customCheckAppRunning
+  Var /GLOBAL CmdPath
+  Var /GLOBAL FindPath
+  Var /GLOBAL PowerShellPath
+  !ifmacrodef customCheckAppRunning
+    !insertmacro customCheckAppRunning
+  !else
+    StrCpy $CmdPath "$SYSDIR\cmd.exe"
+    StrCpy $FindPath "$SYSDIR\find.exe"
+    StrCpy $PowerShellPath "$SYSDIR\WindowsPowerShell\v1.0\powershell.exe"
```

Comment:
@mmaietta You can edit this to `StrCpy $PowerShellPath "blablablaexe"` if you want "disable" powershell.